taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -452,38 +483,77 @@ def train_action(parameters, graph_config, input, task_group_id, task_id):
 
     parameters = dict(parameters)
 
-    start_stage = input.pop("start-stage", None)
-    if start_stage:
-        if "previous_group_ids" not in input:
-            raise Exception(
-                "'previous_group_ids' is required to use 'start-stage' (otherwise we can't skip earlier tasks)"
-            )
+    previous_group_ids = input.pop("previous-group-ids", None)
+    if previous_group_ids:
+        # Resume the pipeline by reusing the completed tasks from previous task groups
+        import taskcluster
+
+        TC_MOZILLA = "https://firefox-ci-tc.services.mozilla.com"
```

Comment:
You should be able to find this in `TASKCLUSTER_ROOT_URL` in the environment rather than hardcoding it.